scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -261,22 +261,21 @@ def build_graph(df, dataset):
 
     # couple of histograms specific for tests with fakes
     df = df.Define("deltaPhiMuonMet", "std::abs(wrem::deltaPhi(goodMuons_phi0,MET_corr_rec_phi))")
-    mTStudyForFakes = df.HistoBoost("mTStudyForFakes", mTStudyForFakes_axes, ["goodMuons_eta0", "goodMuons_pt0", "goodMuons_charge0", "transverseMass", "passIso", "hasCleanJet", "deltaPhiMuonMet", "nominal_weight"])
+    mTStudyForFakes = df.HistoBoost("mTStudyForFakes", mTStudyForFakes_axes, ["goodMuons_eta0", "goodMuons_pt0", "goodMuons_charge0", "transverseMass", "passIso", "hasCleanJet", "deltaPhiMuonMet", "nominal_weight"], storage=hist.storage.Double())
```

Comment:
Sure, I removed the explicit storage type of Double again. I also added a flag "aufiliary_histograms" which is by default true but set to false in case unfolding is run but to save memory.